e2e/testdata/quick-start/build.envd

```diff
@@ -4,3 +4,5 @@ def build():
         "numpy",
     ])
     shell("zsh")
+
```

Comment:
I guess these two blank lines are generated by https://github.com/tensorchord/envd/blob/9e35cb8b4ab30e1aee638b88df3ef579a0154bd6/e2e/bytecode_hash_test.go#L25

Maybe we can find a better way later. So running e2e test locally won't modify the test files.